packages/gatsby-plugin-google-analytics/src/index.js

```diff
@@ -53,4 +53,32 @@ OutboundLink.propTypes = {
   onClick: PropTypes.func,
 }
 
-export { OutboundLink }
+/**
+ * This allows the user to create custom events within their Gatsby projects.
+ * @param       {string} category Required - The object that was interacted with (e.g.video)
```

Comment:
No spaces in between please
```suggestion
 * @param {string} category Required - The object that was interacted with (e.g.video)
```

Reply:
You miss the other lines also